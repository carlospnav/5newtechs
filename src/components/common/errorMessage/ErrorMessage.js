import styled from "styled-components"
import css from "theme/variables"

export default styled.span`
  font-family: ${css.font.openSans};
  font-weight: ${css.fontWeight.semibold};
  font-size: 14px;
  line-height: 18px;
  color: ${css.color.valencia};
`
