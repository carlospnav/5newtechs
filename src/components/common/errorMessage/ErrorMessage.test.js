import { shallow } from "enzyme"
import ErrorMessage from "./ErrorMessage"
import "jest-styled-components"

describe("<ErrorMessage>", () => {
  const wrapper = shallow(<ErrorMessage>This is an error!</ErrorMessage>)
  it("Should render an ErrorMessage properly.", () => {
    expect(wrapper).toMatchSnapshot()
  })
})