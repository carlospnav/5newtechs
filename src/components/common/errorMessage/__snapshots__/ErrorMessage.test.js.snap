// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorMessage> Should render an ErrorMessage properly. 1`] = `
.c0 {
  font-weight: 600;
  font-size: 14px;
  line-height: 18px;
  color: #D74B4B;
}

<span
  className="c0"
>
  This is an error!
</span>
`;
